open-metadata-implementation/governance-servers/data-engine-proxy-services/data-engine-proxy-services-server/src/main/java/org/odpi/openmetadata/governanceservers/dataengineproxy/processor/DataEngineProxyService.java

```diff
@@ -153,6 +160,29 @@ public void run() {
 
     }
 
+    public void runWithReports() {
```

Comment:
Lets call this method load()

I do not like mentioning Reports at this level since it is too specific for the implementation. (example: different technologies may not have this concept of lineage reports). 

load() method is responsible for orchestrating the calls of the underlying connector implementation and leaves to the connector to determine how that is going to be done (as is now just that the name implies already solution)